shaping/heroic_roses.py

```diff
@@ -0,0 +1,44 @@
+# This Python script writes the heroic_roses.yaml file for Klee.
+
+def write_yaml(f, shapes):
+    f.write("dimensions: 2\n\nshapes:\n")
+
+    for s in shapes:
+        n = s["num"]
+        for i in range(n):
+            f.write(f"- name: {s['mat']}{i}\n")
+            f.write(f"  material: {s['mat']}\n")
+            if s["dnr"]:
+                lststr = str(s["dnr"]).replace("'",'"')
+                f.write(f"  does_not_replace: {lststr}\n")
+            f.write(f"  geometry:\n")
+            f.write(f"    format: c2c\n")
+            fn = s["file"]
+            if "%d" in fn:
+                fn = fn % i
+            f.write(f"    path: ../contours/heroic_roses/{fn}\n")
+
+def main():
+    shapes = ({"mat":"greenishbrown","num": 1, "file":"greenishbrown.contour", "dnr":[]},
+          {"mat":"brightgreen","num": 1, "file":"brightgreen.contour", "dnr":["greenishbrown"]},
+          {"mat":"darkgreen","num": 6, "file":"darkgreen%d.contour", "dnr":[]},
+          {"mat":"blue","num": 7, "file":"blue%d.contour", "dnr":[]},
+          {"mat":"yellow","num": 3, "file":"yellow%d.contour", "dnr":[]},
+          {"mat":"purplish","num": 2, "file":"purplish%d.contour", "dnr":[]},
+          {"mat":"pink","num": 2, "file":"pink%d.contour", "dnr":["darkgreen"]},
+          # NOTE: swap red/black when replacement rules work.
```

Comment:
Do the replacement rules work sufficiently in https://github.com/LLNL/axom/pull/1003 to resolve this comment?

Reply:
I removed the comment since it was obsolete. It turned out that this dataset was only useful with the SamplingShaper, which already implemented replacement rules. I did not know that when I wrote the comment.